xml_forms.module

```diff
@@ -21,3 +23,46 @@ function xml_forms_get_schema($filename) {
     drupal_not_found();
   }
 }
+
+/**
+ * Implements hook_help().
+ */
+function xml_forms_help($route) {
+  switch ($route) {
+    // Main help page for the module.
+    case 'help.page.xml_forms':
+      // About section.
+      $output = '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('The XML Forms group of modules includes:</br>' .
+        '<ul>' .
+        '<li>Islandora XML Forms <a href=":xml_forms">wiki page*</a></li>' .
+        '<li>Islandora XML Form Builder <a href=":xml_form_builder">wiki page*</a></li>' .
+        '<li>Islandora XML Form Elements <a href=":xml_form_elements">wiki page*</a></li>' .
+        '<li>Islandora XML Form API <a href=":xml_form_api">wiki page*</a></li>' .
+        '<li>Islandora XML Schema API <a href=":xml_schema_api">wiki page*</a></li>' .
```

Comment:
Could use something to separate the items from the links... colons? Move to a definition list?